lualib/resty/events/worker.lua

```diff
@@ -331,6 +335,8 @@ function _M:publish(target, source, event, data)
     assert(type(source) == "string" and source ~= "", "source is required")
     assert(type(event) == "string" and event ~= "", "event is required")
 
+    log(DEBUG, "source=", source, ", event=", event, ", data=", cjson_encode(data))
```

Comment:
Please add some prefix to the log message.

Reply:
done.